docs/developer_docs.rst

```diff
@@ -183,6 +183,7 @@ Import the bodhi2 database
 
     curl -O https://infrastructure.fedoraproject.org/infra/db-dumps/bodhi2.dump.xz
     sudo -u postgres createdb bodhi2
+    sudo -u postgres bash -c "psql -c \"create role bodhi2;\""
```

Comment:
Thanks for the contribution! Is the call to bash needed, or could this line just be:

```
sudo -u postgres psql -c "create role bodhi2;"
```